### DISCUSSION TITLE
Carx street STUCK at loading

### DISCUSSION THREAD

**Discussion by Easonxsy** [PlayCover/PlayCover#1572] [Q&A]
<img width="1440" alt="Screenshot 2024-07-08 at 1 58 01 PM" src="https://github.com/PlayCover/PlayCover/assets/91364442/b0a7dd19-4220-4b16-b65d-a9c022a02bf3">
can recreate by install carx street ipa downloaded from decrypt.day into playcover then luaching the app

**Comment by Takaowers** [comment]
U found solution??
